.circleci/config.yml

```diff
@@ -341,10 +357,10 @@ jobs:
           keys:
               - tox-cache-django-{{ checksum "tox.ini" }}
       - run: tox -e '{py27,py34,py35,py36}-django{18,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached' --result-json /tmp/django.1.results
-      - run: tox -e '{py27,py34,py35,py36}-django-autopatch{18,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached' --result-json /tmp/django.2.results
+      - run: export TOX_SKIP_DIST=False; tox -e '{py27,py34,py35,py36}-django-autopatch{18,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached' --result-json /tmp/django.2.results
```

Comment:
I'm not sure if this is shell dependent. But I believe that having `export TOX_SKIP_DIST=False; ...` will set `TOX_SKIP_DIST=False` for the subsequent command. 

However, CircleCI when it executes a `run` job may reconfigure the environment. Do you know if this is the case?

Reply:
So you are perfectly right, CircleCI executes the steps in the same container, without reconfiguring it, so what you are saying make absolute sense. A fix on the way!